pages/table/resizable-columns.page.tsx

```diff
@@ -215,7 +229,9 @@ export default function App() {
           key={renderKey}
           header={<Header>Simple table</Header>}
           stickyHeader={stickyHeader}
-          columnDefinitions={columns.map(col => (withColumnIds ? col : { ...col, id: undefined }))}
+          columnDefinitions={columns
+            .map(col => (withColumnIds ? col : { ...col, id: undefined }))
+            .map(col => ({ ...col, width: percentageWidths ? col.percentageWidth ?? col.width : col.width }))}
```

Comment:
Nit: you are passing the `percentageWidth` attribute here which is not part of the `ColumnDefinition` interface. I guess TypeScript does not complain because of the spread syntax